tests/integration/README.md

```
@@ -0,0 +1,112 @@
+2. `jest.integration.config.js` forces resolution to `<rootDir>/node_modules/better-sqlite3` (the Node version) instead of `release/app/node_modules` (the Electron version).
+
+```javascript
+moduleNameMapper: {
+  // ...
+  '^better-sqlite3$': '<rootDir>/node_modules/better-sqlite3',
+},
+```
+
+## 📂 Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)